docs/topics/mta-web-administrator-view.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+## Administrator view
+The administrator view is intended to be used by administrators to set up the Tackle instance environment.
+![](/Tackle2/Views/AdminMenu.png)
+
+### Credentials
+This management module enables administrators to create and manage credentials for access to private repositories.  It also allows for the architects to assign the credentials to applications without knowing their contents.
+![](/Tackle2/Views/CredMain.png)
+
+The credentials page displays the available credentials with an Edit and Delete buttons and the following fields:
+* Name
+* Description
+* Type
+    * Source Control
+     * Username/Password
+     * Source Private Key/Passphrase
+    * Maven Settings File
+    * Proxy
+* Type specific information
+* Created by
+
+**Note:** Type specific credential information such as keys and passphrases will be hidden or shown as [Encrypted].
+
+### Repositories
+This management module configures the repositories used by Tackle with the following options.
+![](/Tackle2/Views/GitConfig.png)
+* Git
+    * Consume insecure Git repositories
+![](/Tackle2/Views/SubvConfig.png)
+* Subversion
+    * Consume insecure Subversion repositories
+![](/Tackle2/Views/MavenConfig.png)
+* Maven
+    * Manage repository size
+    * Force update of dependencies
```

Comment:
I think forcing the update of Maven deps has been removed. @jortel can you confirm?

Reply:
Yes.  Removed.